res/gamedata/scripts/ui_mm_opt_controls.script

```diff
@@ -0,0 +1,33 @@
+class "opt_controls" (CUIWindow)
```

Comment:
Add original file with separate commit and then commit your changes to it.

Reply:
I did It.